lib/internal/Magento/Framework/Data/Form/Element/AbstractElement.php

```diff
@@ -62,8 +62,6 @@ abstract class AbstractElement extends AbstractForm
     protected $_escaper;
 
     /**
-     * Lock html attribute
-     *
```

Comment:
@SilinMykola perhaps this file should be reverted

Reply:
@andrewbess Hi! Thanks for your advice, I made needed changes.